app/src/main/java/com/voipgrid/vialer/sip/SipService.java

```diff
@@ -159,15 +161,8 @@ public void onCreate() {
 
         mLogger.d("onCreate");
 
-        IntentFilter filter = new IntentFilter();
-        filter.addAction(TelephonyManager.ACTION_PHONE_STATE_CHANGED);
-
-        registerReceiver(phoneStateReceiver, filter);
-
-        IntentFilter networkFilter = new IntentFilter();
-        networkFilter.addAction(ConnectivityManager.CONNECTIVITY_ACTION);
-
-        registerReceiver(mNetworkSwitch, networkFilter);
+        mBroadcastReceiverManager.registerReceiverViaGlobalBroadcastManager(phoneStateReceiver,TelephonyManager.ACTION_PHONE_STATE_CHANGED);
```

Comment:
There should be a space after the comma that's separating the arguments (phoneStateReceiver, TelephonyManager)